src/app/container/tool-file-editor/tool-file-editor.component.ts

```diff
@@ -114,7 +114,7 @@ export class ToolFileEditorComponent extends FileEditing {
           this.toggleEdit();
           this.containerService.setTool(editedDockstoreTool);
           const updatedVersion = this.getNewestVersion(editedDockstoreTool.workflowVersions);
-          this.alertService.detailedSuccess('Saved version ' + updatedVersion.name + ' of hosted entry ' + editedDockstoreTool.namespace);
+          this.alertService.detailedSuccess('Saved version ' + updatedVersion.name + ' of hosted tool ' + editedDockstoreTool.toolname);
```

Comment:
I too am confused by all the different parts of a tool :). @Richard-Hansen can you confirm this is what you're expecting?

Reply:
I think this will print ```Saved version 1 of hosted tool null``` (or throw an error) for cases where toolname is not defined. 

I believe the complete format of an entry is as follows:
```registry/namespace/name/*toolname*```, where toolname is an optional parameter. See the bottom dialogue box.
<img width="586" alt="Screen Shot 2020-12-10 at 2 42 01 PM" src="https://user-images.githubusercontent.com/36607471/101821384-e2b76380-3af5-11eb-8da3-410b93d56ff5.png">

For example, both the following are valid hosted tool names:
```quay.io/richardhansen/myHostedTool```
```quay.io/richardhansen/myHostedTool/myCustomToolName```

I think we would want to print the custom tool name if it is present, but omit it otherwise. So, using a ternary, something along the lines of 
```editedDockstoreTool.toolname ? editedDockstoreTool.name + "/" + editedDockstoreTool.toolname : editedDockstoreTool.name```
 would return only the ```name``` variable if toolname is not truthy, otherwise it would return the string ```name/toolname```.